### ISSUE TITLE
Final Peer Review by tj97

### ISSUE THREAD

**Issue by tajseattle** [kathleenzhu/MedicalNoshow#14]
This project tries to predict which patients will not show up for medical appointments. This will help reduce wastage of medical resources and doctor time. It is an admirable goal.

Things I liked:
1) The section on data analysis is very detailed, with relations of different fields being very clearly analyzed. The correlations between features and predictions are explored and visualized in great detail.
2) Merging the original data with the weather data seemed like a very smart move.
3) Comprehensive Loss function selection and Comprehensive result analysis.

Things I did not like:
1) Weak model selection - Only used linear models and decision trees. It seems unlikely that a complicated phenomenon like no show would be captured by such simple models, and I would encourage using neural nets, SVMs etc. This is reflected in the non-exceptional performance of the model.
2) Cross Validation may help with overfitting
3) Feature Engineering section is disappointingly small, and it is not clear how the features were made.
4) The report has a lot of unnecessary details about models and loss functions that can be omitted since only ORIE 4741 students are reviewing these.
